src/tests/test_magic_cmd.py

```diff
@@ -319,16 +350,67 @@ def test_table_schema_profile(ip, tmp_empty):
             assert cell == str(expected[profile_metric][0])
 
 
-def test_table_profile_warnings_styles(ip, tmp_empty):
-    ip.run_cell(
+def test_table_schema_profile_dbapi(ip_with_dbapi, tmp_empty):
+    sqlite_conn_b = sqlite3.connect("b.db")
+    ip_with_dbapi.push({"sqlite_conn_b": sqlite_conn_b})
+
+    ip_with_dbapi.run_cell(
+        """%%sql sqlite_conn_b
+CREATE TABLE t (n FLOAT);
+INSERT INTO t VALUES (11);
+INSERT INTO t VALUES (22);
+INSERT INTO t VALUES (33);
+"""
+    )
+
+    ip_with_dbapi.run_cell(
         """
-    %%sql sqlite://
+    %%sql sqlite_conn_b
+    ATTACH DATABASE 'b.db' AS b_schema;
+    """
+    )
+
+    expected = {
+        "count": ["3"],
+        "mean": ["22.0000"],
+        "min": ["11.0"],
+        "max": ["33.0"],
+        "std": ["11.0000"],
+        "unique": ["3"],
+        "freq": [math.nan],
+        "top": [math.nan],
+    }
+
+    out = ip_with_dbapi.run_cell("%sqlcmd profile -t t --schema b_schema").result
```

Comment:
when writing tests, it's better to provide the full name of each argument (`--table` instead of `-t`)

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/785/files#diff-9188e75e824219dd191cdc62d6d19f2b15c4bd0cf564abe3646ea907c32cd61dR387)